algorithms/unit_tests/TestStdAlgorithmsCopyIf.cpp

```diff
@@ -220,13 +220,13 @@ void verify_data(const std::string& name, ViewTypeFrom view_from,
     std::size_t count = 0;
     for (std::size_t i = 0; i < view_from_h.extent(0); ++i) {
       if (pred(view_from_h(i))) {
-        EXPECT_TRUE(view_test_h(count++) == view_from_h(i));
+        EXPECT_EQ(view_test_h(count++), view_from_h(i));
```

Comment:
I know this is in the original code, but it is bad to have side effect code (`count++`) inside macros.

(Unless I missed another one, this appears to be the only occurrence.)

Reply:
05d532023a65680ef0bf0c86a22138cfbf94e334